### DISCUSSION TITLE
Disregard messages if several come in rapid succession?

### DISCUSSION THREAD

**Discussion by allibear17** [karashiiro/TextToTalk#78] [Ideas]
TTS does not handle well messages that are typically macros of several chat messages, meant to make some sort of ascii art.  If it's possible to recognize that the same person sent several messages within a second of each other, you could disregard reading those?

**Comment by johnysandels** [comment]
One thing you could do is create exclusions in the plugin settings for common ASCII art, so it won't read messages that includes those symbols

**Comment by karashiiro** [comment]
I could see about implementing this - converting to issue.

**Comment by karashiiro** [comment]
https://github.com/karashiiro/TextToTalk/issues/79
